datafusion/core/src/physical_plan/joins/sort_merge_join.rs

```diff
@@ -295,6 +321,65 @@ impl ExecutionPlan for SortMergeJoinExec {
         )
     }
 
+    fn ordering_equivalence_properties(&self) -> OrderingEquivalenceProperties {
+        let mut new_properties = OrderingEquivalenceProperties::new(self.schema());
```

Comment:
Since `SymmetricHashJoin` doesn't maintain input ordering under any circumstance (since it has alternating build sides), its ordering_equivalence implementation should be empty.